evelyn_chen/alpha_challenges/conditionals.md

```diff
@@ -27,13 +43,24 @@ Create a Ruby method called `temperature_status` that takes three arguments: the
   - If the air conditioner is non-functional and the current temperature is below the the desired temperature, print the string "Fix the A/C whenever you have the chance. It's cool."
 
 ```ruby
-# write your code here
+def temperature_status(current_temp, AC_functional, desired_temp)
+  print 'Turn on the A/C, please.' if AC_functional && current_temp > desired_temp
+  print 'Fix the A/C now! It is hot!' if current_temp > desired_temp && !AC_functional
+  print 'Fix the A/C whenever you have the chance. It is cool.' if current_temp < desired_temp && !AC_functional
+  end
```

Comment:
Gotta tie up your loose ends 😉 (I'm proud of this pun so don't come for me)

Reply:
Lol, thank you for catching that!

Reply:
Wait, nevermind, sorry, I thought I forgot `end`

Reply:
Nope, you have an extra one 😄

Reply:
Oh, oops, thank you!